static/html/5xx.html

```diff
@@ -8,34 +8,27 @@
     <!-- NB: Relies on the fact that PipelineCachedStorage also includes
          a copy of the file with no hash in the name.
          This file will not load in dev unless you set DEBUG = False. -->
-    <link href="/static/min/portico.css" rel="stylesheet">
+    <link href="/static/styles/portico.css" rel="stylesheet">
```

Comment:
I assume we'll want to undo this hunk?  Maybe move it into its own "for testing" commit?